README.md

```diff
@@ -1,83 +1,98 @@
 # website-redesign
 
-This is the organizational repository for the Node.js Website Redesign
-Strategic Initiative, as defined under the [Community Committee].
-In plain speak, this initiative is tasked with organizing and facilitating
-the redesign of the https://nodejs.org website, and keeping track of
-cross-cutting concerns with other groups such as the Internationalization WG.
+This is the organizational repository for the Node.js Website Redesign Strategic
+Initiative, as defined under the [Community Committee]. In plain speak, this
+initiative is tasked with organizing and facilitating the redesign of the
+https://nodejs.org website, and keeping track of cross-cutting concerns with
+other groups such as the Internationalization WG.
 
 ## Governance
 
 This initiative adopts the general Node.js [Code of Conduct], as well as its own
-[contributing guidelines].
+[Contributing Guidelines].
 
 ## Meetings
 
-Meeting notes can be accessed [here](./meetings). Future meetings can be listed using [this link](https://github.com/nodejs/website-redesign/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+Meeting+in%3Atitle).
+[Meeting notes] from past meetings are maintained for later reference.
```

Comment:
why was the ref to the meetings dir and issues removed from here :thinking: ??

Reply:
Removed inline links in favour of affixed links — had to reword text in meetings section to fit this in. As a bonus, this made the links more semantic.